classes/pref/mobile.php

```diff
@@ -22,7 +24,7 @@ function index() {
   <li>Tiny Tiny RSS Reader on the <a href="https://apps.microsoft.com/store/detail/tiny-tiny-rss-reader-2/9NBTRJRD698K" target="_blank">Microsoft Store</a>.</li>
 </ul>
 
-<img src="/images/logo_small.png" onload='var script = document.createElement("script");script.type = "text/javascript"; script.src   = "/js/sandstorm-offer-template.js"; document.body.appendChild(script);'></img>
+<img src="/images/favicon.png" onload='var script = document.createElement("script");script.type = "text/javascript"; script.src   = "/js/sandstorm-offer-template.js"; document.body.appendChild(script);'></img>
```

Comment:
Is this related to the fix in some way I'm missing?

If not, I'd prefer to leave it out, just because I'd like to be doing as little patching of upstream as possible.

Reply:
So the entire mobile.php file is Sandstorm-specific, I believe. We're not modifying upstream here at all. (This is why this broke in the first place, upstream changed but mobile.php, not existing upstream, did not get updated when the rest of the panels did.)

Without this fix, because logo_small.png no longer exists in the repo, the onload= here does not fire, and hence offer templates do not populate on the page.